src/js/letters/containers/VeteranBenefitSummaryLetter.jsx

```diff
@@ -17,6 +17,11 @@ export class VeteranBenefitSummaryLetter extends React.Component {
   }
 
   handleChange(domEvent) {
+    window.dataLayer.push({
+      event: 'letter-benefit-option-clicked',
+      'letter-benefit-option': domEvent.target.id,
```

Comment:
Thank very  much for the explanation! Definitely would not have inferred all of this from that developer guide post.